src/main.js

```diff
@@ -1,17 +1,30 @@
-var savePosterBtn = document.querySelector('.save-poster');
-var showSavedBtn = document.querySelector('.show-saved');
-var showRandomBtn = document.querySelector('.show-random');
-var showFormBtn = document.querySelector('.show-form');
-var makePosterBtn = document.querySelector('.make-poster');
-var showMainBtn = document.querySelector('.show-main');
-var backToMainBtn = document.querySelector('.back-to-main');
-var imageInput = document.querySelector('#poster-image-url');
+// query selector variables go here 👇
+
+//main page
+var posterImage = document.querySelector('.poster-img');
+var posterTitle = document.querySelector('.poster-title');
+var posterQuote = document.querySelector('.poster-quote');
+var randomButton = document.querySelector('.show-random');
+var mainPosterPage = document.querySelector('.main-poster');
+var currentPoster = document.querySelector('.poster');
+var savePosterButton = document.querySelector('.save-poster');
+var showSavedPostersButton = document.querySelector('.show-saved');
+
+//make your own poster page
+var posterForm = document.querySelector('.poster-form');
+var makeYourOwnPosterButton = document.querySelector('.show-form');
+var showMyPosterButton = document.querySelector('.make-poster');
+var imageUrlInput = document.querySelector('#poster-image-url');
 var titleInput = document.querySelector('#poster-title');
 var quoteInput = document.querySelector('#poster-quote');
-var mainPosterSection = document.querySelector('.main-poster');
-var posterFormSection = document.querySelector('.poster-form');
-var savedPostersSection = document.querySelector('.saved-posters');
+var takeMeBackButton = document.querySelector('.show-main');
+var backToMainButton = document.querySelector('.back-to-main');
+
+// saved posters page
+var savedPostersForm = document.querySelector('.saved-posters');
 var savedPostersGrid = document.querySelector('.saved-posters-grid');
+
+// we've provided you with some data to work with 👇
```

Comment:
Does this data need to live in `main.js`? Or could it be a separate file e.g. `data.js`?